-------------------------------------------------------------------------------
Test set: com.adaptive.quizapp.AdaptiveQuizApplicationTests
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 15.628 s <<< FAILURE! - in com.adaptive.quizapp.AdaptiveQuizApplicationTests
deleteUserTest  Time elapsed: 0.04 s  <<< FAILURE!
org.mockito.exceptions.verification.WantedButNotInvoked: 

Wanted but not invoked:
userRepository bean.delete(
    com.adaptive.quizapp.entity.User@71c16f67
);
-> at com.adaptive.quizapp.AdaptiveQuizApplicationTests.deleteUserTest(AdaptiveQuizApplicationTests.java:40)

However, there was exactly 1 interaction with this mock:
userRepository bean.deleteById(0);
-> at com.adaptive.quizapp.service.UserServiceImpl.deleteStudentById(UserServiceImpl.java:82)


	at com.adaptive.quizapp.AdaptiveQuizApplicationTests.deleteUserTest(AdaptiveQuizApplicationTests.java:40)

